### PR TITLE
Upgrade for Mongo base image to mongo:4.2.3-bionic

### PR DESCRIPTION
**Pull request by venkata-subbareddyK**
Snyk reporting as many as CVE's with older versions of base images
so upgrading to latest available to fix some of them

Signed-off-by: Koppolu <venkata.subbareddy.koppolu@intel.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -10,4 +10,4 @@ require (
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 )
 
-go 1.13
+go 1.14
```
Comment (tsconn23) [inline go.mod]: Supporting Go 1.13 for Geneva release

File: cmd/Dockerfile [hunk 2]
```diff
@@ -16,7 +16,7 @@
 ###############################################################################
 # Mongo DB image for EdgeX Foundry
 
-FROM golang:1.13-alpine AS builder
+FROM golang:1.14-alpine AS builder
```
Comment (tsconn23) [inline cmd/Dockerfile]: We are supporting Go 1.13 for Geneva release

**Comment by venkata-subbareddyK** [comment]
> Thank you @venkata-subbareddyK
> As you recently joined the DevOps WG, we welcome your contribution.
> 
> Please note: This project has not yet made the decisions to move to go 1.14
> I would request the change back to the 1.13. The Snyk report for the latest version of mongo:4.2.3-bionic addresses the High Severity CVEs so that change is approved.
> https://snyk.io/test/docker/mongo:4.2.3-bionic/
> 
> I will ask other approvers from SIR team to review as well.

Thank you James, i have updated the PR by reversing to golang:1.13

File: go.mod (hunk 1 above)
Comment (venkata-subbareddyK) [reply go.mod]: @tsconn23 Thank you for the review, i have reverted to golang:1.13

**Comment by venkata-subbareddyK** [comment]
Current status of the project supports golang:1.13, so reverting to supported version

**Comment by venkata-subbareddyK** [comment]
> LGTM
> Could update the PR message to just "Upgrade for Mongo base image to mongo:4.2.3-bionic"

Updated

**Comment by difince** [comment]
@venkata-subbareddyK please rebase

**Comment by venkata-subbareddyK** [comment]
> @venkata-subbareddyK please rebase

sure @difince

**Comment by venkata-subbareddyK** [comment]
@difince Thank you

**Comment by venkata-subbareddyK** [comment]
Thank you @tingyuz

**Pull request merged**